packages/actionmenu/src/react/menu.js

```diff
@@ -16,7 +17,11 @@ const slide = css.keyframes(
 const styles = {
   menu: ({ _isNested, origin }) => {
     return compose(
-      css(stylesheet['.psds-actionmenu']({ slide })),
+      css(stylesheet['.psds-actionmenu']),
+      css({
+        animation: `${slide} ${motion.speedNormal} forwards`
+      }),
+      css(stylesheet['@keyframes psds-actionmenu__keyframes__slide']),
```

Comment:
What does including this line here accomplish?

Reply:
`'.psds-actionmenu': ({ slide }) => ({   ` does not build css with the class `'.psds-actionmenu`. That enables both the react code to function and the css to build correctly. I can try and fix the build process but it doesn't properly handle the examples you posted above.

Reply:
Huh, that's unexpected.  I would have thought that the usage of `glamor.css.keyframes` on line 13 and the application of that on line 22 in `animation` would have been enough to make the React component work.  And then the CSS build doesn't use this file at all, just `css/index.js`.